tests/monitoring/vm_monitoring.go

```diff
@@ -81,6 +83,28 @@ var _ = Describe("[Serial][sig-monitoring]VM Monitoring", Serial, decorators.Sig
 		})
 	})
 
+	Context("VMI metrics", func() {
+		It("should have kubevirt_vmi_phase_transition_time_seconds buckets correctly configured", func() {
+			vmi := libvmifact.NewGuestless()
+			tests.RunVMIAndExpectLaunch(vmi, 240)
+
+			for _, bucket := range virtcontroller.PhaseTransitionTimeBuckets() {
+				labels := map[string]string{"le": strconv.FormatFloat(bucket, 'f', -1, 64)}
+
+				GinkgoLogr.Info("Checking bucket", "labels", labels)
```

Comment:
@machadovilaca Is this print necessary? we usually try to avoid verbosity in tests unless it really needed. From past experience we know that people are frustrated to analyze failed e2e tests because of plenty of printouts.